client/signup/steps/user/index.jsx

```diff
@@ -52,13 +51,21 @@ export default React.createClass( {
 	},
 
 	submitForm( form, userData, analyticsData ) {
+		var abtest = require( 'lib/abtest' ).abtest;
 		let flowName = this.props.flowName,
 			queryArgs = {};
 
 		if ( this.props.queryObject && this.props.queryObject.jetpack_redirect ) {
 			queryArgs.jetpackRedirect = this.props.queryObject.jetpack_redirect;
 		}
 
+		if ( abtest( 'coldStartReader' ) === 'noEmailColdStart' ) {
```

Comment:
Can we add a comment explaining why we're using these settings for this particular test?